url_matcher/util.py

```diff
@@ -0,0 +1,25 @@
+from functools import lru_cache
+from urllib.parse import urlparse
+
+from tldextract import tldextract
+
+
+@lru_cache(100)
+def get_domain(url):
```

Comment:
Let's add some annotations for consistency: `def get_domain(url: str) -> str:`

Same comment goes for the function next to this.

Reply:
Good catch. I missed a few. Done here: https://github.com/zytedata/url-matcher/pull/2/commits/d27bf80b2fe304cb2a64ce5ae2cb601658f86316